usaspending_api/common/matview_manager.py

```diff
@@ -32,14 +32,6 @@
                 "sql_filename": "mv_agency_office_autocomplete.sql",
             },
         ),
-        (
-            "summary_state_view",
-            {
-                "model": mv.SummaryStateView,
-                "json_filepath": str(JSON_DIR / "summary_state_view.json"),
-                "sql_filename": "summary_state_view.sql",
-            },
-        ),
```

Comment:
@collinwr will this require a change to the nightly pipeline config? I know we have the SummaryStateView matview files being generated as part of the pipeline, but it doesn't look like we're using those .sql files. I don't think this is going to break anything in the pipeline but just wanted a second set of eyes to confirm that.

Reply:
As a note, a couple of files that should be cleaned up in the config repo that reference the summary_state_view matview:
- matview_runner.groovy
  - `MATVIEW_LIST`
- parameterized_nightly_pipeline_docker_shared.groovy
  - `OPTIONAL_MATVIEWS`

Reply:
I believe `summary_state_view.json` can also be removed if we are removing traces of this being a matview.